skins/cat17/src/app/lib/form_components.js

```diff
@@ -200,6 +201,16 @@ module.exports = {
 		bankDataElements.accountNumberElement.on( 'change', createDefaultChangeHandler( store, 'accountNumber' ) );
 		bankDataElements.bankCodeElement.on( 'change', createDefaultChangeHandler( store, 'bankCode' ) );
 		return component;
+	},
+
+	makeTextComponentMoreSnappy: function( textComponent, debounceDelay ) {
```

Comment:
I get the point, but maybe `createEagerlyValidatingTextComponent` would be a better fit.